.github/workflows/ci-conan.yml

```diff
@@ -38,8 +38,8 @@ jobs:
           - name: Install packages
             run: |
                 sudo apt-get install -y gcc g++ cmake wget git python3 python3-pip python3-venv ccache
-                pip install "conan>=1.51.0,<2.0.0"
-                conan profile new --detect default && conan profile update settings.compiler.libcxx=libstdc++11 default
+                pip install "conan==2.8.0"
+                conan profile detect
```

Comment:
don't we need to set an explicit profile name here?

Reply:
not needed anymore